src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/data/SetVariableDataType.java

```diff
@@ -0,0 +1,13 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2.data;
+
+/**
+ * Defines the type used in the ChangeConfigurationSRequest.
```

Comment:
"ChangeConfiguration**S**Request" spelling mistake ?

Reply:
Oh yeah my bad, it should be SetVariablesRequest...

Reply:
Deleted